TipCatDotNet.Api/Services/Payments/PayoutService.cs

```diff
@@ -0,0 +1,94 @@
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using CSharpFunctionalExtensions;
+using Microsoft.EntityFrameworkCore;
+using Stripe;
+using TipCatDotNet.Api.Data;
+
+namespace TipCatDotNet.Api.Services.Payments;
+
+public class PayoutService : IPayoutService
+{
+    public PayoutService(AetherDbContext context, Stripe.PayoutService payoutService)
+    {
+        _context = context;
+        _payoutService = payoutService;
+    }
+
+
+    public Task<Result> PayoutAll(CancellationToken cancellationToken = default)
+    {
+        return Result.Success()
+            .Bind(PayoutByUsers);
+
+
+        async Task<Result> PayoutByUsers()
+        {
+            Result result = Result.Success();
+            var allStripeAccounts = await _context.StripeAccounts
```

Comment:
We could get only accounts with pending payments

Reply:
Payment Intent and Payout they are different flows. Only succeeded payment intents fall into available balance to payout.

Reply:
I had saw it later in the review, but we can't spam Stripe anyway